packages/manager/apps/telecom/src/app/telecom/pack/move/resume/move-resume.html

```diff
@@ -251,6 +224,18 @@ <h3 data-translate="pack_move_doing"></h3>
                     data-translate-values="{ duration: $ctrl.PROMO_DISPLAY[$ctrl.offer.selected.offer.prices.promotion.subscription.duration].value }"
                     data-ng-if="$ctrl.offer.selected.offer.prices.promotion"
                 ></p>
+                <p
+                    data-ng-bind-html="$ctrl.displayPrice('pack_move_resume_install_fees', $ctrl.offer.selected.offer.prices.installFees.price.text)"
+                    data-ng-if="!$ctrl.offer.selected.offer.prices.promotion && $ctrl.offer.selected.offer.prices.installFees.price"
+                ></p>
+                <p
+                    data-ng-bind-html="$ctrl.displayPrice('pack_move_resume_creation_line_fees', $ctrl.offer.selected.offer.prices.creationLineFees.price.text)"
+                    data-ng-if="$ctrl.offer.selected.offer.prices.creationLineFees.price"
+                ></p>
+                <p
+                    data-ng-bind-html="$ctrl.displayPrice('pack_move_resume_comfort_activation', $ctrl.offer.selected.offer.prices.gtrComfortFees.price.text)"
+                    data-ng-if="$ctrl.offer.selected.offer.gtrComfortActivated"
+                ></p>
```

Comment:
**nitpick:** `translate` directive can be used directly used here.

Reply:
I made a function because with the `class` argument in the span, the data-translate don't work.

Reply:
```suggestion
                <p
                    data-translate="pack_move_resume_install_fees"
                    data-translate-values="{
                         price: '<span class="text-price"> {{$ctrl.offer.selected.offer.prices.installFees.price.text}}</span>
                    }"
                    data-translate-compile
                    data-ng-if="!$ctrl.offer.selected.offer.prices.promotion && $ctrl.offer.selected.offer.prices.installFees.price"
                ></p>
```

Reply:
doesn't work :
```
 50.00 €' }" data-translate-compile data-ng-if="!$ctrl.offer.selected.offer.prices.promotion && $ctrl.offer.selected.offer.prices.installFees.price" >

' }" data-translate-compile data-ng-if="$ctrl.offer.selected.offer.prices.creationLineFees.price" >

250.00 €' }" data-translate-compile data-ng-if="$ctrl.offer.selected.offer.gtrComfortActivated" >

Un réengagement de 12 mois débutera à la date de facturation.

9.99 }" data-translate-compile data-ng-if="$ctrl.offer.selected.offer.needNewModem && $ctrl.offer.selected.shipping.mode === 'transporter'" >

392.99 € }" data-translate-compile class="font-weight-bold" >
```